.travis.yml

```diff
@@ -8,6 +8,9 @@ os:
   - linux
 matrix:
   include:
+    - env:
+        - REACT_APP_ENV='production'
+      if: tag IS present
```

Comment:
I'm not sure about this change, as this was recently removed. See https://github.com/gnosis/safe-react/commit/8057ebfbaed8df7b6ea324d9eeb4f9d3eeba093f#diff-354f30a63fb0907d4ad57269548329e3

Reply:
Yes, this lines of code should be removed. Correction is already on the file